### ISSUE TITLE
Work in other web extensions

### ISSUE THREAD

**Issue by photodow** [carbon-design-system/devtools#82]
#### The problem
So there have been a couple of small edge cases where the web extension doesn't work. One example of this I've seen is in other web extensions. The IBM Accessibility tool for instance has been wanting to use Carbon devtools, but at least to date I haven't found a way to allow injection of it into another web extension like theirs.

> ##### Potential solution
> One potential solution I've thought about on this would be to remove some of the dependencies on web extension core features as much as possible. Like the pop-up, and even some of the background features. We could potentially render the UI inside of page the itself off to the side or something. In addition to that, it would allow us to create a kind of stand-alone app that you could inject into your own pages either through a CDN, or NPM package. Similar to how Firebug did it back in the day just to help out with these kinds of extreme edge cases. There maybe a better way to do this, but I wanted to document this thought before I forgot it.
